pontoon/checks/utils.py

```diff
@@ -8,17 +8,13 @@
 from pontoon.checks.libraries import run_checks
 
 
-def get_translations(**qs_filters):
+def prefetch_translations():
```

Comment:
Wouldn't it be more elegant to join both of these functions into a `TranslationQuerySet` method, e.g. `for_checks()`, which would prefetch relevant data and filter only translations of `DB_FORMATS`?

```python
def for_checks(self):
    """
    Prefetch data with fields needed for running checks.
    """
    return self.prefetch_related(
        'entity__resource__entities',
        'locale',
    ).filter(
        entity__resource__format__in=DB_FORMATS,
    )
```

Instead of:
```python
prefetched_translations = prefetch_translations().filter(pk__in=translations)
```

You would then call (note that we only care about `DB_FORMATS` checks on batch actions, too):
```python
translations = translations.for_checks()
```

Reply:
That was one of my initial ideas, but I decided to skip this approach because that could later end-up with circular dependencies between `base` and `checks` modules (e.g. because of DB_FORMATS). However, It's not a strong opinion -> I can adjust this patch if you prefer to create `for_checks()` method in the `base` module.

Reply:
As agreed on IRC, we'll go ahead and see if the circular dependency indeed causes trouble.